site/en/blog/new-in-chrome-90/index.md

```diff
@@ -0,0 +1,277 @@
+
+[Shadow DOM][shadow-dom], part of the Web Components standard, provides a way
+to scope CSS styles to a specific DOM subtree and isolate that subtree from
+the rest of the document. Until now, the only way to use Shadow DOM was to
+construct a shadow root using JavaScript.
+
+```js
+const host = document.getElementById('host');
+const opts = {mode: 'open'};
+const shadowRoot = host.attachShadow(opts);
+const html = '<h1>Hello Shadow DOM</h1>';
+shadowRoot.innerHTML = html;
+```
+
+This works fine for client-side rendering, but not so well in server side
+rendering where there is no built in way to express Shadow Roots in the server
+generated HTML. But, starting in Chrome 90, with the Declarative Shadow DOM,
+you're good to go. You can create shadow roots using only HTML.
+
+A Declarative Shadow Root is a `<template>` element with a `shadowroot`
+attribute. It's detected by the HTML parser and immediately applied as the
+shadow root of its parent element.
+
+```html
+<host-element>
+  <template shadowroot="open">
+    <slot></slot>
+  </template>
+  <h2>Light content</h2>
+</host-element>
+```
+
+Loading the pure HTML markup results in this DOM tree.
+
+```html
+<host-element>
+  #shadow-root (open)
+  <slot>
+    ↳
+    <h2>Light content</h2>
+  </slot>
+</host-element>
+```
+
+This gives us the benefits of Shadow DOM's encapsulation and slot projection
+in static HTML. No JavaScript is needed to produce the entire tree, including
+the Shadow Root.
+
+Check out [Declarative Shadow DOM][dec-sha-dom] on web.dev for more details.
+
+## And more {: #more }
+
+And of course there's plenty more.
+
+To help improve privacy, and even loading speeds for users visiting sites
+that support HTTPS, Chrome's [address bar will use `https://` by default][https-by-default].
+And if you haven't set up an automatic redirect from HTTP to HTTPS, now would
+be a great time to do that.
+
+And an [AV1 encoder][av1-encoder] is shipping in Chrome desktop that is
+specifically optimized for video conferencing with WebRTC integration.
+
+## Further reading
+
+This covers only some of the key highlights. Check the links below for
+additional changes in Chrome 90.
+
+* [What's new in Chrome DevTools (90)](/blog/new-in-devtools-90/)
+* [Chrome 90 deprecations & removals](/blog/deps-rems-90/)
+* [ChromeStatus.com updates for Chrome 90](https://www.chromestatus.com/features#milestone%3D90)
+* [What's new in JavaScript in Chrome 90](https://v8.dev/blog/v8-release-90)
+* [Chromium source repository change list](https://chromium.googlesource.com/chromium/src/+log/89.0.4389.75..90.0.4430.71)
+
+## Subscribe
+
+Want to stay up to date with our videos, then [subscribe](https://goo.gl/6FP1a5)
+to our [Chrome Developers YouTube channel](https://www.youtube.com/user/ChromeDevelopers/),
+and you'll get an email notification whenever we launch a new video.
+
+I'm Pete LePage, and as soon as Chrome 91 is released, I'll be right here to
+tell you what's new in Chrome!
```

Comment:
```suggestion
tell you what's new in Chrome.
```

Reply:
I'd like to keep it for the emphasis... When you watch the video, you'll see me get a little funky at the end. :)

Reply:
I'll approve this. Please be aware that exclamation marks are not for emphasis. They're for exclamations. "Stop!" "Wow!" "Great job". Some things in speech just don't translate to text.